### ISSUE TITLE
flag provided but not defined: -pluginsDir

### ISSUE THREAD

**Issue by Fank** [grafana/grafana#17613]
**What happened**:
I updated 6.2.2 -> 6.2.3 and now the container won't start because of:
```
flag provided but not defined: -pluginsDir
Usage of grafana-cli:
  -test.bench regexp
    	run only benchmarks matching regexp
  -test.benchmem
    	print memory allocations for benchmarks
  -test.benchtime d
    	run each benchmark for duration d (default 1s)
  -test.blockprofile file
    	write a goroutine blocking profile to file
  -test.blockprofilerate rate
    	set blocking profile rate (see runtime.SetBlockProfileRate) (default 1)
  -test.count n
    	run tests and benchmarks n times (default 1)
  -test.coverprofile file
    	write a coverage profile to file
  -test.cpu list
    	comma-separated list of cpu counts to run each test with
  -test.cpuprofile file
    	write a cpu profile to file
  -test.failfast
    	do not start new tests after the first test failure
  -test.list regexp
    	list tests, examples, and benchmarks matching regexp then exit
  -test.memprofile file
    	write an allocation profile to file
  -test.memprofilerate rate
    	set memory allocation profiling rate (see runtime.MemProfileRate)
  -test.mutexprofile string
    	write a mutex contention profile to the named file after execution
  -test.mutexprofilefraction int
    	if >= 0, calls runtime.SetMutexProfileFraction() (default 1)
  -test.outputdir dir
    	write profiles to dir
  -test.parallel n
    	run at most n tests in parallel (default 1)
  -test.run regexp
    	run only tests and examples matching regexp
  -test.short
    	run smaller test suite to save time
  -test.testlogfile file
    	write test action log to file (for use only by cmd/go)
  -test.timeout d
    	panic test binary after duration d (default 0, timeout disabled)
  -test.trace file
    	write an execution trace to file
  -test.v
    	verbose: print additional output
```

**What you expected to happen**:
A normal startup like in the update 6.2.1 -> 6.2.2

**How to reproduce it (as minimally and precisely as possible)**:

helm: https://github.com/helm/charts/tree/master/stable/grafana
helm values:
```
image:
  tag: 6.2.3
deploymentStrategy: Recreate
ingress:
  enabled: true
  annotations:
    kubernetes.io/ingress.class: traefik
    kubernetes.io/tls-acme: "true"
  hosts:
    - grafana.example.com
  resources:
    limits:
      memory: 200Mi
      cpu: 250m
    requests:
      memory: 50Mi
      cpu: 50m
persistence:
  enabled: true
  storageClassName: standard
  accessModes:
    - ReadWriteOnce
  size: 1Gi
adminUser: "admin"
plugins:
  - grafana-worldmap-panel
  - grafana-clock-panel
  - grafana-piechart-panel
  - grafana-polystat-panel
  - jdbranham-diagram-panel
  - bessler-pictureit-panel
  - btplc-peak-report-panel
  - zuburqan-parity-report-panel
  - natel-plotly-panel
  - scadavis-synoptic-panel
  - marcuscalidus-svg-panel
grafana.ini:
  server:
    root_url: https://grafana.example.com
  users:
    allow_sign_up: false
  auth.anonymous:
    enabled: false
  auth.basic:
    enabled: true
```

**Anything else we need to know?**:
https://github.com/grafana/grafana/blob/master/packaging/docker/run.sh#L75

**Environment**:
- Grafana version: 6.2.3
- Data source type & version:
- OS Grafana is installed on: docker
- User OS & Browser:
- Grafana plugins:
  - grafana-worldmap-panel
  - grafana-clock-panel
  - grafana-piechart-panel
  - grafana-polystat-panel
  - jdbranham-diagram-panel
  - bessler-pictureit-panel
  - btplc-peak-report-panel
  - zuburqan-parity-report-panel
  - natel-plotly-panel
  - scadavis-synoptic-panel
  - marcuscalidus-svg-panel
- Others:

**Comment by Bregor** [comment]
6.2.2:

    $ docker run -it --rm -e GF_INSTALL_PLUGINS="grafana-piechart-panel" grafana/grafana:6.2.2
    installing grafana-piechart-panel @ 1.3.6
    from url: https://grafana.com/api/plugins/grafana-piechart-panel/versions/1.3.6/download
    into: /var/lib/grafana/plugins
    
    ✔ Installed grafana-piechart-panel successfully

6.2.3:

    $ docker run -it --rm -e GF_INSTALL_PLUGINS="grafana-piechart-panel" grafana/grafana:6.2.3
    flag provided but not defined: -pluginsDir

**Comment by Fank** [comment]
@gotjosh i pinged you because you created the latest release, don't know which else to ping :D

**Comment by Bregor** [comment]
https://github.com/grafana/grafana/pull/17606/files#diff-1146cad1c61c159a07758ade40c6ac2aR21
Looks like `flag.Parse` is executing in the wrong place now.

**Comment by marefr** [comment]
Thanks for reports. We've made this our highest priority to fix, but we won't be able to release a fix (v6.2.4)  until tomorrow. Because of this we decided to update docker hub so that `latest` tag points to 6.2.2 and 6.2.3 tag has been removed.
